bluemira/equilibria/optimisation/problem/_breakdown.py

```diff
@@ -201,49 +201,55 @@ def __init__(
         self.eq = breakdown
         self.opt_algorithm = opt_algorithm
         self.opt_conditions = opt_conditions
+        self.bounds = self.get_current_bounds(self.coilset, max_currents, self.scale)
 
         self._args = {
             "c_psi_mat": np.array(
                 coilset.psi_response(*breakdown_strategy.breakdown_point)
             ),
             "scale": self.scale,
         }
+
         x_zone, z_zone = breakdown_strategy.calculate_zone_points(n_B_stray_points)
         stray_field_cons = FieldConstraints(
             x_zone, z_zone, B_max=B_stray_max, tolerance=B_stray_con_tol
         )
-
         self._constraints = constraints
         if self._constraints is not None:
             self._constraints.append(stray_field_cons)
         else:
             self._constraints = [stray_field_cons]
 
-        max_currents = np.atleast_1d(max_currents)
-        self.bounds = (-max_currents / self.scale, max_currents / self.scale)
```

Comment:
Could you comment on this removal?

Reply:
Not removed, just moved. Do for consistency between the COP's